src/app/shared/constants.ts

```diff
@@ -29,6 +29,7 @@ export const includesAuthors = 'authors';
 export const bootstrap4smallModalSize = '300px';
 export const bootstrap4mediumModalSize = '500px';
 export const bootstrap4largeModalSize = '800px';
+export const bootstrap4extraLargeModalSize = '1140px';
```

Comment:
Note: I'm using a larger modal size here (https://getbootstrap.com/docs/4.2/components/modal/#optional-sizes) because `bootstrap4largeModalSize` is slightly too small to display the table without scrolling.